<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="controlLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnSendMessage">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Отправить сообщение</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="edtMsg">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QToolButton" name="btnRefresh">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icons/refresh.png</normaloff>icons/refresh.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Автоотправка</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnSendPhoto">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Отправить выбранное фото</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="photosListWidget">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="cameraGrid"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Главная</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="separator"/>
    <addaction name="actionHistory"/>
    <addaction name="actionClearPhotos"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="actionOneCol"/>
    <addaction name="actionTwoCol"/>
    <addaction name="actionThreeCol"/>
    <addaction name="actionFourCol"/>
    <addaction name="actionFiveCol"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="actionAddCamera"/>
    <addaction name="actionManageCameras"/>
    <addaction name="actionSystemSettings"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAboutUs"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Экспорт настроек</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Импорт настроек</string>
   </property>
  </action>
  <action name="actionAddCamera">
   <property name="text">
    <string>Добавить камеру</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="actionAboutUs">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionManageCameras">
   <property name="text">
    <string>Управление камерами</string>
   </property>
  </action>
  <action name="actionOneCol">
   <property name="text">
    <string>Один столбец</string>
   </property>
  </action>
  <action name="actionTwoCol">
   <property name="text">
    <string>Два столбца</string>
   </property>
  </action>
  <action name="actionThreeCol">
   <property name="text">
    <string>Три столбца</string>
   </property>
  </action>
  <action name="actionFourCol">
   <property name="text">
    <string>Четыре столбца</string>
   </property>
  </action>
  <action name="actionFiveCol">
   <property name="text">
    <string>Пять столбцов</string>
   </property>
  </action>
  <action name="actionSystemSettings">
   <property name="text">
    <string>Настройки системы</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionClearPhotos">
   <property name="text">
    <string>Очистить базу фотографий</string>
   </property>
  </action>
  <action name="actionHistory">
   <property name="text">
    <string>История реагирвоания</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
